data/convert_Pfam-C2tab.py

```diff
@@ -0,0 +1,31 @@
+
+path = '/Users/mland/my_KBASE/kb_ObjectInfo/data/'
+
+cat = 'Other'
+
+outfile = open(path + "Pfam-C.clans.v35.tsv","w")
```

Comment:
should use the "with" form to open the outfile.

so it would be 
with open(...) as outfile:
  with open(...) as file1: